swift/vapor/Dockerfile

```diff
@@ -6,4 +6,4 @@ COPY Package.swift main.swift ./
 
 RUN swift build -c release
 
-CMD .build/release/server serve -e prod -h 0.0.0.0 -p 3000
+CMD .build/release/server serve --hostname 0.0.0.0 --port 3000 --env prod
```

Comment:
oops, that should have been `-H` (capital). The change you made works, too. Thanks!

Reply:
FWIW, you could write this as:
```
CMD .build/release/server serve -b 0.0.0.0:3000 -e prod
```